zetta_utils/layer/volumetric/index.py

```diff
@@ -308,5 +308,17 @@ def contains(self: VolumetricIndex, point: Sequence[float]) -> bool:
         """
         return self.bbox.contains(point, self.resolution)
 
+    def line_intersects(
+        self: VolumetricIndex, pointA: Sequence[float], pointB: Sequence[float]
```

Comment:
Could you change this to snake case to conform to the rest of the codebase? Like `point_a` and `point_b` or `point1` and `point2`?

Reply:
Fixed!